contrib/src/java/org/apache/hadoop/hive/contrib/serde2/RegexSerDe.java

```diff
@@ -76,6 +76,8 @@
     RegexSerDe.INPUT_REGEX_CASE_SENSITIVE })
 public class RegexSerDe extends AbstractEncodingAwareSerDe {
 
+  public static final Logger log = LoggerFactory.getLogger(RegexSerDe.class.getName());
```

Comment:
Since we are introducing a `static final` logger in every class there is no reason to make it public. The intention is to use them inside each class so every logger should be private.

Also according to the naming conventions in the Java specification (https://docs.oracle.com/javase/specs/jls/se20/html/jls-6.html#jls-6.1) constants should be all uppercase.

Reply:
yes makes sense I will change this to private